examples/OECC/config/rc.local

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+#
+
+export DLC=/psc/dlc
```

Comment:
Given this is in `rc.local` I presume it means you're setting a standard install location so that everything else can assume DLC is in this location? If so, there are still several scripts which explicitly set DLC to the `/psc/dlc` location. If the user decides to install elsewhere (perhaps using the default offered by the OpenEdge installer) would they know to change this?

Reply:
The demo scripts work as a prescriptive demo environment.
The demo scripts install OpenEdge at a known location.
Using a custom installation is outside of the scope of the demo scripts. This can be done if desired.

Reply:
I didn't think of that being a prescriptive environment and a set path because it wasn't mentioned clearly. Assume nothing and that users are uninformed and lazy. Case in point, I don't know any of this stuff and didn't think about the install locations.